models/comments.model.js

```diff
@@ -0,0 +1,28 @@
+const db = require("../db/connection.js")
+
+exports.deleteCommentByIdOnComments = (req) => {
+    const commentID = req.params.comment_id.toString();
+    if(commentID.match(/^\d+$/)) {      
```

Comment:
when using the binding of `$1` psql will throw an error for you if the data type is not valid ( i.e. not a number )